interface/usergroup/user_admin.php

```diff
@@ -167,43 +167,46 @@
 
 <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
 <script type="text/javascript" src="../../library/dialog.js"></script>
-<script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-1-9-1/index.js"></script>
 <script type="text/javascript" src="../../library/js/common.js"></script>
 
 <script src="checkpwd_validation.js" type="text/javascript"></script>
 
+<!-- validation library -->
+<?php  require_once($GLOBALS['srcdir'] . "/validation/validation_script.js.php"); ?>
+<?php  require_once($GLOBALS['srcdir'] . "/validation/validate_core.php"); ?>
+<?php
+//Gets validation rules from Page Validation list.
+//Note that for technical reasons, we are bypassing the standard validateUsingPageRules() call.
```

Comment:
Out of curiosity what is technical reason why can't use validateUsingPageRules() here?